frontend/assets/stylesheets/components/_action.scss

```diff
@@ -125,11 +125,10 @@
 .action--booking {
     color: $white;
     background-color: $black;
-    border-color: $black;
+    border: 0 none;
```

Comment:
Indeed. I got into a weird habit of writing the longer version, trying to break it but I seem to be lapsing…